pkgs/applications/networking/cisco-packet-tracer/customRequireFile.nix

```diff
@@ -0,0 +1,28 @@
+{ requireFile }:
+
+{ filename
+, version
+, alternativeURL
+, sha256
+}: requireFile {
+  name = filename;
+  message = ''
+    Unfortunately, we cannot download the file ${filename} automatically.
+    You have two known options to get this file:
+    - Download the deb file from the NetAcad website: http://www.netacad.com/
+      - You should download the exact same version (${version}) used in this package if it's not the exact version it will not pass.
+      - With the deb file you can import it to the nix store using the following command:
+
+      nix-store --add-fixed sha256 ${filename}
+
+    - Download from Internet Archive: It works for direct downloads but the problem is that it's slow to download even on fast Internet. You can download directly using the following command:
```

Comment:
Internet Archive is not the official means of getting this software. I suggest you remove it.